Formula/rsync.rb

```diff
@@ -2,7 +2,7 @@ class Rsync < Formula
   desc "Utility that provides fast incremental file transfer"
   homepage "https://rsync.samba.org/"
   url "https://rsync.samba.org/ftp/rsync/rsync-3.1.3.tar.gz"
-  mirror "https://mirrors.kernel.org/gentoo/distfiles/rsync-3.1.3.tar.gz"
+  mirror "https://deb.debian.org/gentoo/distfiles/rsync-3.1.3.tar.gz"
```

Comment:
Oh, this was also a false positive by the auditor. The Debian mirrors won't have Gentoo packages. I'll back this change out.

Reply:
Not a false positive by the audit (I [looked to raise a PR to fix it, but it's fine](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/rubocops/urls.rb#L133-L139)), just an overly liberal find and replace on my end when making these changes. 😳